java/performance/pom.xml

```diff
@@ -169,10 +173,17 @@
                         <argument>${benchmark.filter}</argument>
                         <argument>-f</argument>
                         <argument>${benchmark.forks}</argument>
+                        <argument>-jvmArgs</argument>
+                        <argument>${benchmark.jvmargs}</argument>
                         <argument>-wi</argument>
                         <argument>${benchmark.warmups}</argument>
                         <argument>-i</argument>
                         <argument>${benchmark.runs}</argument>
+                        <argument>${benchmark.list}</argument>
+                        <argument>-rff</argument>
+                        <argument>${benchmark.resultfile}</argument>
+                        <argument>-rf</argument>
```

Comment:
@liyafan82 Here is a change from #8245 . Since I cannot find to add `-rf json` or not to add it in `<arguments>`, the current implementation always generates `jmh-result.json` and change a file name if it will be overridden.

I would like to update here if there is a selection method to add `-rf json` or not to add it.

Reply:
It looks good. Thanks.